include/toolchain/gcc.h

```diff
@@ -462,5 +462,21 @@ do {                                                                    \
 		_value_a_ < _value_b_ ? _value_a_ : _value_b_; \
 	})
 
+/**
+ * @brief Calculate power of two ceiling for some value
+ *
+ * @param x Unsigned long value
```

Comment:
Though this macro was already here, it is worth to document that if x == 0 the result is undefined.

Reply:
agreed. On GCC it actually return 1 as expected, but this is UB. I'll update the docs.